src/PHPMailer.php

```diff
@@ -362,7 +364,7 @@ class PHPMailer
      *
      * @see SMTP::$Debugoutput
      *
-     * @var string|callable|\Psr\Log\LoggerInterface
+     * @var string|callable|LoggerInterface
```

Comment:
Since this is optional, I prefer having this left as a FQCN; it's clearer where it comes from at the point of use.

Reply:
rewritten the commit